cypher/cypher-docs/src/docs/dev/syntax/expressions.asciidoc

```diff
@@ -78,3 +84,30 @@ Any number literal may contain an underscore `_` between digits.
 There may be an underscore between the `0x` or `0o` and the digits for hexadecimal and octal literals.
 
 include::../ql/query-syntax-case.adoc[leveloffset=+1]
+include::../ql/query-syntax-label.adoc[leveloffset=+1]
+
+=== List of Label Expressions examples and what they match
+In the following table, a green tick is shown if the `Label Expression` would match the node.
+
+[%header, cols="2h,1,1,1,1,1,1,1,1"]
+|===
+|
+8+^|Node
+1.1+|Label expression|()|(:A)|(:B)|(:C)|(:A:B)|(:A:C)|(:B:C)|(:A:B:C)
+1.1+|()|✅|✅|✅|✅|✅|✅|✅|✅
+1.1+|(:A)||✅|||✅|✅||✅
+1.1+|(:A&B)|||||✅|||✅
+1.1+|(:A\|B)||✅|✅||✅|✅|✅|✅
+1.1+|(:!A)|✅||✅|✅|||✅|
+1.1+|(:!!A)||✅|||✅|✅||✅
+1.1+|(:A&!A)||||||||
+1.1+|(:A\|!A)|✅|✅|✅|✅|✅|✅|✅|✅
+1.1+|(:%)||✅|✅|✅|✅|✅|✅|✅
+1.1+|(:!%)|✅|||||||
+1.1+|(:%\|!%)|✅|✅|✅|✅|✅|✅|✅|✅
+1.1+|(:%&!%)||||||||
+1.1+|(:A&%)||✅|||✅|✅||✅
+1.1+|(:A\|%)||✅|✅|✅|✅|✅|✅|✅
+1.1+|(:(A&B)&!(B&C))|||||✅|||
+1.1+|(:!(A&%)&%))|||✅|✅|||✅|
+|===
```

Comment:
I think it would be worth highlighting the fact that the `:A:B:C` syntax cannot be comibned with the new label expression syntax, `:A:B&C` will throw an error